src/SDKs/Search/DataPlane/Microsoft.Azure.Search/generate.cmd

```diff
@@ -0,0 +1,27 @@
+::
```

Comment:
@brjohnstmsft BTW, would be good to know which issues you found, we should resolve those. We've moved a long way since the NuGet versions regarding Swagger features but also *flexibility* of code generation. Would be cool if we had moved everybody over, I'd be glad to help.